pkg/manifest/export_test.go

```diff
@@ -1,3 +1,12 @@
 package manifest
 
-var FindStage = findStage
+var (
+	FindStage               = findStage
+	SubscriptionService     = subscriptionService
+	RunInsightsClientOnBoot = runInsightsClientOnBoot
+	InsightsClientDropin    = insightsClientDropin
+)
+
+type (
+	SubscriptionServiceOptions = subscriptionServiceOptions
+)
```

Comment:
Why not just use `package manifest` in the tests? That way, the unit test will be able to use unexported structs and functions.

Reply:
yeah, I keep flip-flopping on this.  Sometimes it's nice to isolate the things that are being tested by exporting them, other times I think it doesn't give us much.  I think I'm going to revert in this case.

Reply:
Amended!